src/loaders/FileLoader.js

```diff
@@ -30,7 +32,12 @@ class FileLoader extends Loader {
 
 		url = this.manager.resolveURL( url );
 
-		const cached = Cache.get( url );
+		const offset = this.offset;
+		const length = this.length;
+		const isRangeRequest = offset !== null;
+		const key = url + ( isRangeRequest ? `:${offset}-${length}` : '' );
```

Comment:
Please fix linting (spaces in `${ something }`).

Reply:
@mrdoob Are the spaced needed even in `${ }`? I couldn't find it in the coding guide and I can find some lines not having spaces in `${ }`in the core

https://github.com/mrdoob/three.js/blob/abe1dff3c21aea03c4fd67cc10af3ce21f6d7814/src/renderers/webgl/WebGLProgram.js#L19